Gopkg.toml

```diff
@@ -0,0 +1,13 @@
+required = ["k8s.io/code-generator/cmd/client-gen"]
```

Comment:
I am not sure if we only use client-gen. Then how about defaulter-gen/informer-gen and so on?

Reply:
Maybe refer to https://github.com/kubeflow/tf-operator/blob/master/Gopkg.toml#L3

Reply:
yes. "k8s.io/code-generator/cmd/client-gen" comes with clientsets/defaulter/informer-gen.

Reply:
Then LGTM